packages/oauth/src/install-provider.ts

```diff
@@ -330,7 +330,10 @@ export class InstallProvider {
     }
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     const _installOptions: InstallURLOptions = installOptions! || this.installUrlOptions!;
-    this.logger.debug(`Running handleInstallPath() with ${_installOptions}`);
+    if (this.logger.getLevel() <= LogLevel.DEBUG) {
```

Comment:
@filmaj I've added this if condition to avoid the cost to run `JSON.stringify` when the logging level INFO or above. That being said, we may want to simplify the logic rather than the slight efficiency improvement.

Reply:
this is an ignorable cost. removed the if condition